CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 0.15.0
+## Added
+* Support AVM 7 updates:
+  * New opcodes:
+    * base64_decode` ([#418](https://github.com/algorand/pyteal/pull/418))
+    * `block` ([#415](https://github.com/algorand/pyteal/pull/415))
+    * `ed25519verify_bare` ([#426](https://github.com/algorand/pyteal/pull/426))
+    * `json_ref` ([#417](https://github.com/algorand/pyteal/pull/417))
+    * `replace2`, `replace3` ([#413](https://github.com/algorand/pyteal/pull/413))
+    * `sha3_256` ([#425](https://github.com/algorand/pyteal/pull/425))
+    * `vrf_verify` ([#419](https://github.com/algorand/pyteal/pull/419))
+  * `Secp256r1` curve for ECDSA opcodes ([#423](https://github.com/algorand/pyteal/pull/423))
+  * Program page count transaction fields ([#416](https://github.com/algorand/pyteal/pull/416))
```

Comment:
Would https://github.com/algorand/pyteal/pull/412 be a more appropriate PR to link to for this?

Also, I think "Program page transaction field access" or similar would be a more accurate description

Reply:
@jasonpaulos Yes - I inadvertently linked the wrong PR.  Thanks for catching.  Updated via https://github.com/algorand/pyteal/pull/481/commits/47b4f629d911f6bd599a61057879ed3d7e8ed700.